host_os.py

```diff
@@ -31,12 +33,40 @@
     'set-env': setup_environment,
     'build-iso': build_iso,
 }
+PACKAGE_DEPENDENCIES_FILE_PATH = "rpm_requirements.txt"
+MISSING_PACKAGE_DEPENDENCIES_ERROR = 4
+
+def is_package_installed(package_name):
+    """
+    Checks if a RPM package is installed
+    """
+
+    cmd = "rpm -q %s" % package_name
+    try:
+        run_command(cmd, shell=True)
+    except SubprocessError as e:
+        # rpm returns 1 when search string is not found and other non-zero values
+        # if an error occurred
+        #pylint: disable=no-member
+        if e.returncode == 1:
+            return False
+        else:
+            raise
+
+    return True
 
 
 if __name__ == '__main__':
     CONF = config.setup_default_config()
     subcommand = CONF.get('default').get('subcommand')
 
+    package_dependencies = open(PACKAGE_DEPENDENCIES_FILE_PATH).readlines()
+    package_dependencies = [p.strip('\n') for p in package_dependencies]
+    missing_packages = [p for p in package_dependencies if not is_package_installed(p)]
+    if missing_packages:
+        print("Following packages should be installed before running this script: %s" % ", ".join(missing_packages))
+        sys.exit(MISSING_PACKAGE_DEPENDENCIES_ERROR)
+
```

Comment:
What about fitting that code in 79 columns, as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)?

```python
pkg_deps = open(PKG_DEPS_FILE).readlines()
pkg_deps = [p.strip('\n') for p in pkg_deps]
missing_pkgs = [p for p in pkg_deps if not is_package_installed(p)]
if missing_pkgs:
    print("Following packages should be installed before running "
          "this script: %s" % ", ".join(missing_pkgs))
    sys.exit(MISSING_PKG_DEPS_ERROR)
```

With shorter variable name you can still have meaningful names with enhanced readbility.